src/game-data/prompts/PromptState.ts

```diff
@@ -62,9 +63,11 @@ export interface ChooseFieldPromptState {
     kind:
         | "oneClick" // plant; buy/sell field
         | "harvest" // multi-field selection
-        | "uproot"; // multi-vine selection
+        | "uproot" // multi-vine selection
+        | "switch"; // two-vine swap
     disabledReasons: Record<FieldId, string | undefined>;
     numSelections: number;
+    submitDisabledReason?: (vines: VineInField[]) => string | undefined;
```

Comment:
If this is only used for `switch` can we name it something more obviously so? Even better would be to split out a different prompt state type (same as how `ChooseWinePromptState` and `FillOrderPromptState` are different even though they're both handled by the `ChooseWinePrompt`)